docs/accelerators.md

```diff
@@ -147,15 +159,15 @@ The order of devices will define their priority, in this case making `device_1`
 
 Here is a config example using heterogeneous plugin with GPU as the primary device and CPU as a fallback.
 
-```json
-{"model_config_list": [
+```bash
+echo '{"model_config_list": [
    {"config": {
       "name": "resnet",
-      "base_path": "/opt/ml/resnet",
+      "base_path": "/opt/model",
       "batch_size": "1",
       "target_device": "HETERO:GPU,CPU"}
    }]
-}
+}' >> models/public/resnet-50-tf/config.json
```

Comment:
We do not recommend users to read documentation on GitHub, but rather link directly to https://docs.openvino.ai/latest/ovms_what_is_openvino_model_server.html

It uses sphinx, which likes to not match correctly with all markdown syntax causing page to not look good on https://docs.openvino.ai/.

Have you tried running sphinx doc build to check if everything looks ok there?

Reply:
doc build and checked
http://openvino-doc.iotg.sclab.intel.com/cvs-78730-new_documentation_test_2/ovms_what_is_openvino_model_server.html